### ISSUE TITLE
Build errer without clean in linux

### ISSUE THREAD

**Issue by hseok-oh** [dotnet/corefx#14791]
When I build managed code in Ubuntu14.04 without clean, I get build fail with error messages
```
/nfs/rbp3/corefx/build.proj(203,5): error : Exact package 'NETStandard.Library2 2.0.0-beta-24801-0' was not restored: found ''

/nfs/rbp3/corefx/build.proj(203,5): error : Exact package 'NETStandard.Library2 2.0.0-beta-24801-0' was not restored: found ''
Command execution failed with exit code 1.
```

This error comes from `project.lock.json`  generated in `System.Security.Cryptography.Xml`.
To avoid this error, we need to build after clean, or option with /p:RestoreDuringBuild=false.

**Comment by hseok-oh** [comment]
This error come from `layout/netstandard20/project.lock.json`.
This generated json file have no `libraries` property and only have `dependencies` property.
```
{
  "frameworks": {
    "netstandard1.7": {
      "dependencies": {
        "NETStandard.Library2": "2.0.0-beta-24801-0"
      }
    }
  }
}
```

**Comment by hseok-oh** [comment]
Typo: NETStandard.Library2 -> NetStandard.Library2 in `layout/netstandard20/project.json`

**Comment by hseok-oh** [comment]
Solution
1. Merge #14887
2. clean `$HOME/.nuget` directory once
3. `./clean.sh -all` once

**Comment by hseok-oh** [comment]
fixed y #14887
